test/testcafe/framework/page-objects/IdentityPageObject.js

```diff
@@ -83,7 +83,7 @@ export default class IdentityPageObject extends BasePageObject {
   }
 
   async hasShowTogglePasswordIcon() {
-    return await Selector('[data-se="password-toggle"]').count > 0;
```

Comment:
is the selector breaking it or is the change to make it explicitly boolean breaking it? if it's the former, will it still work with `Selector('.password-toggle').count > 0`? i'd prefer the explicit boolean return value than rely on JS coercion

Reply:
Good point, but I'm just reverting since we'll want to minimize conflicts on the v2 page object anyway.